### ISSUE TITLE
Accessibility tests should not be susceptible to network/service failures

### ISSUE THREAD

**Issue by dcwarwick** [carbon-design-system/ibm-cloud-cognitive#451]
## What will this achieve?

Currently, the accessibility test matcher `toBeAccessible.js` causes outbound network connections to be made, and fails if those cannot complete successfully. For example, we have sometimes seen this error from the CI check:
![image](https://user-images.githubusercontent.com/6385315/111329323-f3ccbf00-8666-11eb-8515-b5d0ac5710f2.png)
and running the tests with network disabled produces this error:
![image](https://user-images.githubusercontent.com/6385315/111329404-03e49e80-8667-11eb-87a7-4f93651acb75.png)

If an outbound connection is needed, the test should fail gracefully if it cannot be made: the error should be caught and handled with some other informational/warning state but not a test fail.

Preferably, an outbound connection should not be needed. If there is a way to hold a local cache of the remote service/data that is being fetched, that would be better. Investigation needed.

## How will success be measured?

All tests including accessibility tests run successfully even with no network enabled.
